### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by gilesvangruisen** [jadsonlourenco/react-native-shake-event#26]
Howdy!! Thanks for this project. Right now I'm having to clone from master (which is breaking our tests) because a release hasn't yet been cut since merging the `mainQueueSetup` fix. Any chance we could get a new release soon? :) 

Thanks!

**Comment by jadsonlourenco** [comment]
@gilesvangruisen Yes, I'm working on the new release. 

So sorry the delay guys, I'm not using React Native for a long time, yesterday I've started a new project to test and fix this package and release to you.

So, new brach: https://github.com/jadsonlourenco/react-native-shake-event/tree/dev
Using RN 0.5+. I've tested on iOS and tvOS (simulator and real devices), but don't work on android simulator.

### I need help for:
- **Android:** if is possible to detect the shake event "on began" and "on ended", like iOS does. Otherwise we can use the "onEnded" method for Android.
- **TypeScript/Flow:** if someone that use TypeScript and Flow can help to see if need updates.

Just send PR to this branch and I will merge, then I will release a new version on NPM.
Thank you!

**Comment by steliyan** [comment]
I am experiencing some issues using this package and cocoapods, more specifically, when I build the project, I get:
`RNShakeEvent.h:1:9: 'React/RCTBridgeModule.h' file not found`

Have you actually tested with the latest version react native (v.0.57.2) and cocoapods?

**Comment by jadsonlourenco** [comment]
@steliyan are you using the **dev** branch, for now?

**Comment by steliyan** [comment]
@jadsonlourenco yep, going to use the `dev`, seems to be working as expected for now. If I find any issues, I'll keep you posted.

**Comment by jadsonlourenco** [comment]
Ok, thank you.

I'm trying to fix the issue with Android to release a new version.

**Comment by jadsonlourenco** [comment]
Ok guys, so sorry the delay, but I'm trying to figure out the issues on Android to have this module working. I've never used it on Android, I'm learning.

So, until now I discover that I need expose a **@ReactMethod** on Java to the module be loaded on **NativeModules** from JS.

But the shake event don't handle, even if I use the `import com.facebook.react.common.ShakeDetector;`, I mean on the JAVA side. But it work to show the *devMenu*.

So, anyone with Android knowledge to help? Thank you.
